Directory.Packages.props

```diff
@@ -15,6 +15,10 @@
     <!-- The last package version of MSBuild that works with net5.0 is 16.11.0.  Our assemblies are still compiled against MSBuild assembly version 15.1.0.0 which will work with all versions -->
     <MicrosoftBuildVersion Condition="'$(TargetFramework)' == 'netcoreapp5.0'">16.11.0</MicrosoftBuildVersion>
 
+    <!-- Default System.Security.Cryptography.Xml version -->
+    <SystemSecurityCryptographyXml Condition="'$(SystemSecurityCryptographyXml)' == ''">6.0.1</SystemSecurityCryptographyXml>
+    <SystemSecurityCryptographyXml Condition="'$(MicrosoftBuildVersion)' == '16.8.0' Or '$(MicrosoftBuildVersion)' == '16.11.0'">4.7.1</SystemSecurityCryptographyXml>
```

Comment:
The condition on MicrosoftBuildVersion is an interesting trick to make it so when we upgrade the MSBuild package version, this package will work differently. However, there's no logic on all the PackageReferences for the package, or on the PackageVersion either. In which case I expect there'd be a bunch of restore errors.

For this reason, I think the comment at the top of the block should explain it's a transitive dependency of MSBuild, and can probably be removed when MSBuild is updated to a newer version.